app/client/package.json

```diff
@@ -61,6 +61,8 @@
     "glossary-panel": "github:Eastern-Research-Group/glossary",
     "highcharts": "10.1.0",
     "highcharts-react-official": "3.1.0",
+    "rc-slider": "^10.0.0",
+    "rc-tooltip": "^5.1.1",
```

Comment:
Whoops I didn't realize I got rid of the `package-lock.json` file. 

As for the updates, I thought I addressed that in the `MapMouseEvents.js` file. If I click on a location on the map, it shows the updated values for me. If I move the slider while it's open, though, I do need to close it and open it again. Similarly I did just find that the characteristic group counts aren't refreshed if a row is expanded, so I'll need to address that.

I was trying to avoid the `applyEdits` option, because even just doing updates was causing the graphics to flicker.